dist/lob-api-bundled.yml

```diff
@@ -7693,6 +7693,77 @@ components:
       oneOf:
         - $ref: '#/components/schemas/tmpl_id'
         - $ref: '#/components/schemas/local_file_path'
+    single_redirect_url:
+      type: string
+      description: >-
+        Redirect all mail receipients to a single non-personalized web page. To
+        use this type of redirection, just add a redirect_url with the url where
+        the users should be redirected to after scanning the QR code.
+    customized_redirect_url:
+      type: string
+      description: >-
+        Redirect all mail receipients to a unique web page personalized for each
+        recipient. To use this type of redirection, create an extra column in
+        the [audience
+        file](https://help.lob.com/print-and-mail/building-a-mail-strategy/campaign-or-triggered-sends/campaign-audience-guide)
+        with column header `qr_code_redirect_url` and add a unique link against
```

Comment:
I know you have already merged this, but does the column name necessarily need to be named `qr_code_redirect_url`? From [this document](https://docs.google.com/document/d/1-XPb9Gcf15RxldT0HYZ0zzO3DHYgCCuuG6LBbUPKJks/edit), we had to provide something like `"mergeVariableColumnMapping": {  "qr_code_redirect_url": <your column name from the CSV>}`, which would imply the column's name could be whatever.